scm/define-grob-properties.scm

```diff
@@ -1168,15 +1168,16 @@ one below this grob.")
 ;;; w
 ;;;
      (when ,ly:moment? "Global time step associated with this column.")
-     (whiteout ,boolean-or-number? "If a number or true, the grob is
-printed over a white background to white-out underlying material, if
+     (whiteout ,boolean-or-number-or-number-pair? "If a number, number-pair
+     or true, the grob is printed over a white background to white-out underlying material, if
```

Comment:
Latest when this is going into the LilyPond review you'll be asked to keep lines to 72 characters.